packages/terra-markdown/tests/wdio/markdown-spec.js

```diff
@@ -1,18 +1,59 @@
 describe('Markdown', () => {
-  before(() => browser.setViewportSize(Terra.viewports('large')[0]));
+  before(() => browser.setViewportSize({ width: 1000, height: 2000, name: 'large' }));
   describe('Default', () => {
     before(() => browser.url('/#/raw/tests/terra-markdown/markdown/markdown'));
 
-    Terra.should.beAccessible();
-    Terra.should.matchScreenshot();
+    Terra.should.validateElement();
```

Comment:
I went through and added individual tests for the main code themes to catch non passive changes when prismjs updates.